.claude/skills/api-test/SKILL.md

```
@@ -0,0 +1,489 @@
+npm run test:e2e -- tests/e2e/api/budgets.spec.ts
+
+# Run with UI mode for debugging
+npm run test:e2e:ui -- tests/e2e/api/
+
+# Run in headed mode
+npm run test:e2e -- --headed tests/e2e/api/
+```
+
+### 5. Test Organization
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)